src/components/NewCommentForm.tsx

```diff
@@ -14,24 +98,33 @@ export const NewCommentForm: React.FC = () => {
             name="name"
             id="comment-author-name"
             placeholder="Name Surname"
-            className="input is-danger"
+            value={name}
+            onChange={(event) => handleNameChange(event.target.value)}
```

Comment:
Instead of additiona callback get `target.value` in handleNameChange
Fix all similar cases
```suggestion
            onChange={handleNameChange}
```